src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.service.impl.FruitServiceImpl;
+import core.basesyntax.service.impl.ParserServiceImpl;
+import core.basesyntax.service.impl.ReaderServiceImpl;
+import core.basesyntax.service.impl.ReportServiceImpl;
+import core.basesyntax.service.impl.WriteServiceImpl;
+import core.basesyntax.strategy.BalanceOperation;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.PurchaseOperation;
+import core.basesyntax.strategy.ReturnOperation;
+import core.basesyntax.strategy.SupplyOperation;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String INPUT = "src/main/java/resources/input_file.csv";
+    private static final String REPORT = "src/main/java/resources/report_file.csv";
+
+    public static void main(String[] args) {
+        List<String> lines = new ReaderServiceImpl()
+                .readFile(INPUT);
+        Map<FruitTransaction.Operation, OperationHandler> operationHandlerMap = new HashMap<>();
+        operationHandlerMap.put(FruitTransaction.Operation.BALANCE, new BalanceOperation());
+        operationHandlerMap.put(FruitTransaction.Operation.SUPPLY, new SupplyOperation());
+        operationHandlerMap.put(FruitTransaction.Operation.PURCHASE, new PurchaseOperation());
+        operationHandlerMap.put(FruitTransaction.Operation.RETURN, new ReturnOperation());
+        List<FruitTransaction> fruitTransactions = new ParserServiceImpl().parseTransactions(lines);
+        OperationStrategy strategy = new OperationStrategy(operationHandlerMap);
+        FruitService fruitService = new FruitServiceImpl();
+        fruitService.processTransactions(fruitTransactions, strategy);
+        new WriteServiceImpl().writeToFile(REPORT, new ReportServiceImpl().createReport());

```

Comment:
as I remember @rovein advised not to create anonymous instances of the services if you already created variables to be more consistent in your code